src/bilder/images/odl_video_service/files/.env.tmpl

```diff
@@ -65,6 +65,14 @@ VIDEO_S3_SUBTITLE_BUCKET="{{ keyOrDefault "ovs/s3_subtitle_bucket_name" "missing
 VIDEO_S3_THUMBNAIL_BUCKET="{{ keyOrDefault "ovs/s3_thumbnail_bucket_name" "missing_value_s3_thumbnail_bucket_name" }}"
 VIDEO_S3_TRANSCODE_BUCKET="{{ keyOrDefault "ovs/s3_transcode_bucket_name" "missing_value_s3_transcode_bucket_name" }}"
 VIDEO_S3_WATCH_BUCKET="{{ keyOrDefault "ovs/s3_watch_bucket_name" "missing_value_s3_watch_bucket_name" }}"
+OVS_AWS_ROLE_NAME="{{ keyOrDefault "ovs/aws_role_name": ovs_mediaconvert.role.name }}"
+OVS_AWS_ACCOUNT_ID="{{ keyOrDefault "ovs/aws_account_id": aws_account.account_id}}"
+OVS_POST_TRANSCODE_ACTIONS="{{ keyOrDefault "ovs/post_transcode_actions": "cloudsync.api.process_transcode_results" }}"
+OVS_TRANSCODE_ENDPOINT="{{ "ovs/video_s3_transcode_endpoint": transcode_endpoint" }}"
+OVS_VIDEO_TRANSCODE_PREFIX="{{ "ovs/video_s3_upload_prefix": "" }}
+OVS_VIDEO_S3_TRANSCODE_PREFIX="{{ "ovs/video_s3_transcode_prefix": "transcoded" }}"
+OVS_VIDEO_S3_THUMBNAIL_PREFIX="{{ "ovs/video_s3_thumbnail_prefix": "thumbnails" }}"
+OVS_VIDEO_TRANSCODE_QUEUE="{{ "ovs/video_transcode_queue": ovs_mediaconvert.queue.name }}"
```

Comment:
So the format would be

```
VAR_NAME="{{ keyOrDefault "consul-key" "string literal default value" }}"
```
The quotes and spaces are important and you've got colons in all of these which won't work. The first three use `keyOrDefault` but the next five do not. I don't think those that omit it will work at all. They need some kind of function call, I believe. This is (essentially) go-templates. 👞 💥 🔫 

Lines 68, 69, and 71 seems like you're referencing python values from the code below in `__main__.py` rather than string literals defaults. 

Honestly, I like to use "missing_value_<env_var_in_lower_case>" as my string literal default so it is clear that consul-template has failed to find what it seeks. See line 65-67 as examples.